test/services/PlaybookServiceTest.js

```diff
@@ -37,7 +37,10 @@ describe('Playbook Service', function () {
                 dateTimeComparisonValue: '19700101',
                 dataFormat: dateTimeDataFormatValue
             })
-            expect(playbook.GetComparisonValue(getComparisonValueDateTime).isSame('19700101')).to.be.true
+
```

Comment:
I don't see why this revision needs to occur within this propType work. Is this related? If not, please restore to the master version.